librdkafka will fail pkgconf test pipeline

If the pkgconf test pipeline is added to librdkafka it will fail in the following way:

```
2024/11/12 23:12:19 INFO running step "test/pkgconf"
2024/11/12 23:12:19 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:19 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:19 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:19 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:19 INFO 2.3.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 INFO -lrdkafka++ uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + basename /home/build/melange-out/librdkafka-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + dev_pkg=librdkafka-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + apk info -L librdkafka-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + grep -q ^Name: usr/lib/pkgconfig/rdkafka++.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + basename usr/lib/pkgconfig/rdkafka++.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + lib_name=rdkafka++ uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + echo usr/lib/pkgconfig/rdkafka++.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + grep -q '^usr/lib/pkgconfig/rdkafka++.pc$\|^usr/share/pkgconfig/rdkafka++.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + pkgconf --exists rdkafka++ uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + grep -q ^Version: usr/lib/pkgconfig/rdkafka++.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + pkgconf --modversion rdkafka++ uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + grep -q ^Libs: usr/lib/pkgconfig/rdkafka++.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + pkgconf --libs rdkafka++ uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + grep -q ^Cflags: usr/lib/pkgconfig/rdkafka++.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN + pkgconf --cflags rdkafka++ uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN Package zlib was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN Perhaps you should add the directory containing `zlib.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN Package 'zlib', required by 'rdkafka', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN Package 'libcrypto', required by 'rdkafka', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN Package 'libssl', required by 'rdkafka', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN Package 'libzstd', required by 'rdkafka', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:19 WARN Package 'liblz4', required by 'rdkafka', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:20 INFO pod 05e8cf89b7fbfe5bfaf49dd46d7fa3b290a818db631a51340ff758bba4eb15f3 terminated
2024/11/12 23:12:20 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:20 INFO   workspace dir: /home/user/tmp/melange-workspace-2529952248
2024/11/12 23:12:20 INFO   guest dir: /home/user/tmp/melange-guest-3071341479
2024/11/12 23:12:20 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/librdkafka] Error 1
```

--- a/librdkafka.yaml
+++ b/librdkafka.yaml
@@ -1,7 +1,7 @@
 package:
   name: librdkafka
   version: 2.3.0
-  epoch: 3
+  epoch: 4
   description: "The Apache Kafka C/C++ library"
   copyright:
     - license: BSD-2-Clause
@@ -50,7 +50,15 @@ subpackages:
   - name: librdkafka-dev
     pipeline:
       - uses: split/dev
+    dependencies:
+      runtime:
+        - lz4-dev
+        - openssl-dev
+        - zlib-dev
     description: librdkafka dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: false

